packages/framework-core/src/services/read-model-store.ts

```diff
@@ -84,6 +65,55 @@ export class ReadModelStore {
     return joinKey
   }
 
+  private async applyProjectionToReadModel(
+    readModelName: string,
+    readModelID: UUID,
+    projectionMetadata: ProjectionMetadata,
+    entity: EntityInterface
+  ): Promise<unknown> {
+    const readModel = await this.fetchReadModel(readModelName, readModelID)
+    const currentReadModelVersion: number = readModel?.boosterMetadata?.version ?? 0
```

Comment:
We store the `version` of the read model (which matches with the number of times it has been updated)
in the special field `boosterMetadata`